pkg/istio/servicerolebinding/servicerolebinding.go

```diff
@@ -105,6 +108,75 @@ func (srbMgr *ServiceRoleBindingMgr) getSubjects(members []zms.MemberName) []*v1
 	return subjects
 }
 
+// NewConfig returns a new model.Config resource with the given namespace/name and spec
+func NewConfig(namespace string, name string, spec *v1alpha1.ServiceRoleBinding) model.Config {
+
+	meta := model.ConfigMeta{
+		Type:      model.ServiceRoleBinding.Type,
+		Group:     model.ServiceRoleBinding.Group + model.IstioAPIGroupDomain,
+		Version:   model.ServiceRoleBinding.Version,
+		Namespace: namespace,
+		Name:      name,
+	}
+	return model.Config{
+		ConfigMeta: meta,
+		Spec:       spec,
+	}
+}
+
+// parseMemberName parses the Athenz role member into a SPIFFE compliant name
+func parseMemberName(member *zms.RoleMember) (string, error) {
+
+	if member == nil {
+		return "", fmt.Errorf("member is nil")
+	}
+
+	memberStr := string(member.MemberName)
+
+	// special condition: if member == 'user.*', return '*'
+	if memberStr == allUsers {
+		return wildCardAllUsers, nil
+	}
+
+	return common.PrincipalToSPIFFE(memberStr)
+}
+
+// GetServiceRoleBindingSpec returns the ServiceRoleBindingSpec for a given Athenz role and its members
+func GetServiceRoleBindingSpec(roleName string, members []*zms.RoleMember) (*v1alpha1.ServiceRoleBinding, error) {
+
+	subjects := make([]*v1alpha1.Subject, 0)
+	for _, member := range members {
+
+		//TODO: handle member.Expiration for expired members, for now ignore expiration
```

Comment:
Does Athenz themselves remove the expired members?

Reply:
I think ZTS / ZMS does the expiration check when the role members are accessed (during checkAccess call), so ideally we should be doing while we get members of a role during every sync